src/Bicep.LangServer/Deploy/DeploymentHelper.cs

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Bicep.Core;
+using Bicep.Core.Json;
+
+namespace Bicep.LanguageServer.Deploy
+{
+    public class DeploymentHelper
```

Comment:
telemetry?

Reply:
I'll be sending out a follow up PR for telemetry changes. Issue tracking telemetry items: https://github.com/Azure/bicep/issues/6160

Let me know if I am missing something in the above mentioned issue. Thanks!